docs/install/install-from-package/prepare-cache-service.rst

```diff
@@ -50,9 +50,9 @@ Execute the following command to start the service container. The project
 
 .. code-block:: console
 
-   $ cd ${HOME}/halfstack
-   $ docker compose -f docker-compose.hs.redis.yaml -p ${USER} up -d
+   $ cd ${HOME}/halfstack/redis-cluster-default
+   $ docker compose -p ${USER} up -d
```

Comment:
The `-p ${USER}` part is no longer necessary. As the halfstack services are now separated by directory, there's no need to use a project-specific approach. Just the following will be enough for every services:
```shell
cd <specific-service-directory>
docker compose down
docker compose up -d
docker compose logs -f
docker compose ps
...
```